project-base/storefront/components/Pages/PersonalData/Export/PersonalDataExportContent.tsx

```
@@ -52,13 +49,13 @@ export const PersonalDataExportContent: FC<PersonalDataExportContentProps> = ({
-                <h1>{t('Personal data export')}</h1>
-
-                {!!contentSiteText && (
-                    <div className="text-justify" id="personal-data-export-content">
-                        <UserText htmlContent={contentSiteText} />
-                    </div>
-                )}
+                <PageHero
+                    icon={UserIcon}
+                    title={t('Personal data export')}
+                    description={
+                        contentSiteText ? <span dangerouslySetInnerHTML={{ __html: contentSiteText }} /> : undefined
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)